src/wallet/types.ts

```diff
@@ -378,14 +376,17 @@ export interface DecodedOutput {
 
 export interface TxOutput {
   value: OutputValueType;
-  script: string;
+  script: {
+    type: 'Buffer';
+    data: number[];
+  };
```

Comment:
If the wallet-service is sending this "Buffer" type maybe we should change how the wallet-service is serializing the script

Reply:
Hey, we actually receive a simpler `TxInput` and `TxOutput` in websocket transactions, e.g.:

```json
{
  "height": 0,
  "inputs": [
    {
      "address": "H79QdDV5i6KuW3dNJEahkUNe8a9axtHuCu",
      "timelock": null,
      "type": "P2PKH"
    }
  ],
  "nonce": 16746194,
  "outputs": [
    {
      "address": "HDY5QHiE33goHcLxFyJmgxTErwgGHDrLdy",
      "timelock": null,
      "type": "P2PKH"
    }
  ],
  "parents": [],
  "signal_bits": 0,
  "timestamp": 1743779693,
  "token_name": null,
  "token_symbol": null,
  "tx_id": "000009532ec33ba944c5d23b8dc642063539055ca31b4f6d4f534d33651ce8a3",
  "version": 1,
  "voided": false,
  "weight": 16.818364568681208
}
```

Fixed the type and the schema validation